pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -7,8 +7,6 @@
 }:
```

Comment:
I think some of these imports above are no longer needed?

Reply:
removed